raiden-dapp/src/components/InstallBanner.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-banner v-if="visible" color="#1e1e1e" two-line>
+    <template v-if="$vuetify.breakpoint.mobile">
+      {{ $t('install-banner.message-mobile') }}
+    </template>
+
+    <template v-else>
+      {{ $t('install-banner.message-desktop') }}
+    </template>
+
+    <template #actions="{ dismiss }">
+      <v-btn dark text outlined @click="install">
+        {{ $t('install-banner.install') }}
+      </v-btn>
+
+      <v-btn dark text outlined @click="dismiss">
+        {{ $t('install-banner.dismiss') }}
+      </v-btn>
+    </template>
+  </v-banner>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component
+export default class Name extends Vue {
```

Comment:
Should this really be called `Name`?

Reply:
Uuups. Good catch. That is a template from my template to create a Vue component file. xD
I'll fix that!

Reply:
I'm surprised it worked to import it anyways.

Reply:
Why should it? We just import modules as their default export. So `import lalaStupidWhatever from 'serious-library'`. Just works. Remember that `import` statements are just like syntactic suggar. The `require()` function is there maybe more obvious as it is a function call and you store the result into a constant variable. Imports like `import { foo, bar } from 'foobar-module'` are just spreading the returned object. So a module that exports multiple things is just a function that return an object. So the import of `'foobar-module'` returns something like `{ foo: something, bar: something, more: something, ... }`. And we just disassamble it. If we would now use a property that does not exist on this objects (means there is no export with this name), we get an error. But for the default export this is just a new variable we create and we can name it like how ever we want. The name of the default export does not matter.

Reply:
Ok, then it makes sense.

Reply:
:heavy_check_mark: